app/i18n/languages.json

```diff
@@ -8,5 +8,6 @@
     { "value": "ptbr", "iso": "br", "name": "Português Brasileiro", "englishName": "Brazilian Portuguese" },
     { "value": "ru", "iso": "ru", "name": "Русский", "englishName": "Russian" },
     { "value": "zhtw", "iso": "tw", "name": "繁中", "englishName": "Traditional Chinese" },
-    { "value": "zhcn", "iso": "cn", "name": "簡中", "englishName": "Simplified Chinese" }
+    { "value": "zhcn", "iso": "cn", "name": "簡中", "englishName": "Simplified Chinese" },
+    { "value": "ar", "iso": "ar", "name": "عربى", "englishName": "Arabic" }
```

Comment:
This is going to be the wrong flag: `"iso": "ar"` will display the Argentinian flag, see here: http://flag-icon-css.lip.is
There is no flag for Arabic obviously since it's not a country, but maybe you could try "ae" for the UAE?